ivy_tests/test_ivy/test_frontends/test_torch/test_non_linear_activation_functions.py

```diff
@@ -175,3 +175,44 @@ def test_torch_leaky_relu(
         input=np.asarray(x, dtype=input_dtype),
         negative_slope=alpha,
     )
+
+
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(
+                set(ivy_torch.valid_float_dtypes)
+            )
+        ),
+        min_num_dims=1,
+    ),
+    axis=st.integers(-1, 0),
```

Comment:
I think that the possible range of axis is not between -1 and 0 but determined by the dimensioned of the x array generated.

Reply:
I am a little bit confused about how to proceed with the implementation of this. Should I implement a loop logic inside the test_torch_softmin function to account for all axes of the input? Or is there a way to do it in the hypothesis decorator?

(Sorry, I'm new to hypothesis and testing)

Reply:
You can consider adding a helper function to the test file and use it in the given section like this one:
@st.composite
def _dtype_x_bounded_axis(draw, **kwargs):
    dtype, x, shape = draw(helpers.dtype_and_values(**kwargs, ret_shape=True))
    axis = draw(helpers.ints(min_value=0, max_value=len(shape) - 1))
    return dtype, x, axis

Reply:
also, this example here need some small fix as when the shape generated by the helper becomes () representing an empty array x, then the max_value of the int should be set to 0 not -1.